prow/jenkins/jenkins.go

```diff
@@ -161,11 +164,43 @@ func (c *Client) log(methodName string, args ...interface{}) {
 	c.logger.Debugf("%s(%s)", methodName, strings.Join(as, ", "))
 }
 
+// measure records metrics about the provided method, path, and code.
+// start needs to be recorded before doing the request.
+func (c *Client) measure(method, path string, code int, start time.Time) {
+	if c.metrics == nil {
+		return
+	}
+	c.metrics.RequestLatency.WithLabelValues(method, path).Observe(time.Since(start).Seconds())
+	c.metrics.Requests.WithLabelValues(method, path, fmt.Sprintf("%d", code)).Add(1)
+}
+
+// GetSkipMetrics fetches the data found in the provided path. It returns the
+// content of the response or any errors that occured during the request or
+// http errors. Metrics will not be gathered for this request.
+func (c *Client) GetSkipMetrics(path string) ([]byte, error) {
```

Comment:
This func is almost identical to `Get`. Please pull out the common code into a shared helper func.

Reply:
done